pkg/cache/cache.go

```diff
@@ -579,11 +580,14 @@ func (c *Cache) Usage(cqObj *kueue.ClusterQueue) (*ClusterQueueUsageStats, error
 		return nil, errCqNotFound
 	}
 
+	weightedShare, _ := cq.DominantResourceShare()
```

Comment:
if the feature is disabled, we shouldn't spend time calculating this.

Reply:
Good point. Thanks.